build.gradle

```diff
@@ -61,7 +61,7 @@ String getTomcatProject(majorVersion) {
         return ":tomcat-base:tomcat-embedded-9"
     }
 
-    if (majorVersion == 23) {
+    if (majorVersion in [23, 25]) {
```

Comment:
Why not `majorVersion >= 23` ?
or... I think we can even do the `if` on L59 as `majorVersion < 23`, perhaps that's makes it even clearer that embedded is the default from now on.